### PR TITLE
OSDOCS-7970: QE and Style Revisions of MOBB Content for "Configuring the Cluster Log Forwarder for CloudWatch logs and STS"

### PR DESCRIPTION
**Pull request by jneczypor**
[OSDOCS-7970](https://issues.redhat.com//browse/OSDOCS-7970): Full QE review and style revisions of previously migrated MOBB content (" Configuring the Cluster Log Forwarder for CloudWatch logs and STS") in product docs

Version(s):
4.13+

Issue:
https://issues.redhat.com/browse/OSDOCS-7970

Link to docs preview:
https://65560--docspreview.netlify.app/openshift-rosa/latest/cloud_experts_tutorials/cloud-experts-rosa-cloudwatch-sts

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by jneczypor** [comment]
Hi @anpingli! Can I please get a review of this tutorial? I believe you already looked at it when we first merged it, so hopefully it is pretty easy for you. Thank you so much!

**Comment by anpingli** [comment]
Checking now

File: cloud_experts_tutorials/cloud-experts-rosa-cloudwatch-sts.adoc [hunk 1]
```diff
@@ -209,14 +211,14 @@ $ cat << EOF | oc apply -f -
...
+[id="cloud-experts-rosa-cloudwatch-sts-check-aws"]
+== Checking CloudWatch for logs
 
-* Use the AWS console or CLI to validate that there are log streams from the cluster:
+* Use either the AWS console or the AWS CLI to validate that there are log streams from the cluster:
 +
 [NOTE]
 ====
-If this is a fresh cluster, you may not see a log group for `application` logs as there are no applications running yet.
+If this is a new cluster, you may not see a log group for `application` logs as applications are not yet running.
```
Comment (EricPonvelle) [inline cloud_experts_tutorials/cloud-experts-rosa-cloudwatch-sts.adoc:221]: ISG/Vale suggest changing "may" to "might"

File: cloud_experts_tutorials/cloud-experts-rosa-cloudwatch-sts.adoc [hunk 2]
```diff
@@ -49,10 +49,12 @@ $ mkdir -p ${SCRATCH}
 $ echo "Cluster: ${ROSA_CLUSTER_NAME}, Region: ${REGION}, OIDC Endpoint: ${OIDC_ENDPOINT}, AWS Account ID: ${AWS_ACCOUNT_ID}"
 ----
 
-[id="rosa-mobb-cloudwatch-sts-prep-aws"]
-== Prepare AWS Account
+. Ensure all fields output correctly before moving to the next section.
```
Comment (EricPonvelle) [inline cloud_experts_tutorials/cloud-experts-rosa-cloudwatch-sts.adoc:52]: What if you make this an action command. Like:

```
. Ensure all exported environmental variables output correctly before moving to the next section by running the following command:
+
[source,terminal]
----
$ $ echo "Cluster: ${ROSA_CLUSTER_NAME}, Region: ${REGION}, OIDC Endpoint: ${OIDC_ENDPOINT}, AWS Account ID: ${AWS_ACCOUNT_ID}"
----
```

Then, remove that command from the previous box. The `export` commands set these variables, while the `echo` command prints their results.

File: cloud_experts_tutorials/cloud-experts-rosa-cloudwatch-sts.adoc [hunk 3]
```diff
@@ -209,14 +211,14 @@ $ cat << EOF | oc apply -f -
 EOF
 ----
 
-[id="rosa-mobb-cloudwatch-sts-check-aws"]
-== Check AWS CloudWatch for logs
+[id="cloud-experts-rosa-cloudwatch-sts-check-aws"]
+== Checking CloudWatch for logs
 
-* Use the AWS console or CLI to validate that there are log streams from the cluster:
+* Use either the AWS console or the AWS CLI to validate that there are log streams from the cluster:
```
Comment (EricPonvelle) [inline cloud_experts_tutorials/cloud-experts-rosa-cloudwatch-sts.adoc:217]: I think you need one more sentence, like: To validate the logs in the AWS CLI, run the following command:

**Review by EricPonvelle — COMMENTED** [review]
Few small things but otherwise /lgtm

File: cloud_experts_tutorials/cloud-experts-rosa-cloudwatch-sts.adoc [hunk 4]
```diff
@@ -209,23 +215,18 @@ $ cat << EOF | oc apply -f -
...
-====
+* Use either the AWS console or the AWS CLI to validate that there are log streams from the cluster.
+** To validate the logs in the AWS CLI, run the following command:
 +
 [source,terminal]
 ----
 $ aws logs describe-log-groups --log-group-name-prefix rosa-${ROSA_CLUSTER_NAME}
 ----
-+
+
```
Comment (EricPonvelle) [inline cloud_experts_tutorials/cloud-experts-rosa-cloudwatch-sts.adoc:228]: You need the plus here to line up the sample.

**Pull request merged**